.github/crd-config/templates/asciidoctor/operator/gv_list.tpl

```diff
@@ -1,17 +1,13 @@
 {{- define "gvList" -}}
 {{- $groupVersions := . -}}
 
-// Generated documentation. Please do not edit.
-= Redpanda CRD Reference
-:anchor_prefix: k8s-api
-:description: Custom resource definitions for the Redpanda resource. Use the Redpanda resource to create and manage Redpanda clusters with the Redpanda Operator.
-
-{description}
-
-.Packages
-{{- range $groupVersions }}
-- {{ asciidocRenderGVLink . }}
+// Autogenerated documentation. Do not edit this page See https://github.com/redpanda-data/docs/blob/main/.github/workflows/generate-crd.yml.
+{{- with index $groupVersions 0 }}
+= {{ . }}
 {{- end }}
+:anchor_prefix: k8s-api
+:description: Custom resource definitions for Redpanda resources. Use the Redpanda resources to create and manage Redpanda clusters, users and topics with the Redpanda Operator.
+:page-aliases: reference:k-topic-crd.adoc
```

Comment:
I'm not sure if k-topic-crd.adoc is removed in this PR?

Reply:
yes we removed it in this PR.

Reply:
this line adds the redirects from that old page to this one.